core/src/test/java/kafka/testkit/TestKitNode.java

```diff
@@ -28,7 +42,140 @@ default String metadataDirectory() {
         return initialMetaPropertiesEnsemble().metadataLogDir().get();
     }
 
+    default Set<String> logDataDirectories() {
+        return initialMetaPropertiesEnsemble().logDirProps().keySet();
+    }
+
     MetaPropertiesEnsemble initialMetaPropertiesEnsemble();
 
-    boolean combined();
+    Map<String, String> propertyOverrides();
+
+
+    class Builder {
```

Comment:
Do we really need this builder? It seems `TestKitNodes#Builder` is almost same to this builder. Could we move helper method `buildBrokerNode` and `buildControllerNode` to  `TestKitNodes`?